### ISSUE TITLE
Add Bengali language

### ISSUE THREAD

**Issue by Pratik-050** [kubernetes-sigs/headlamp#3270]
## Is your feature request related to a problem? Please describe the impact that the lack of the feature requested is creating.
Add Bengali language so that people who know Bengali can use it easily.

## Describe the solution you'd like
I propose adding support for more languages to Headlamp's UI to make it accessible to a wider audience
. The solution includes the following steps:

Add New Languages like:
Bengali

## What users will benefit from this feature?
There's a large number of native Bengali people, it'll be easier for them to access the UI.

## Are you able to implement this feature?
Yes

**Comment by ChayanDass** [comment]
@Pratik-050   are you currently working on it?  If not, I would like to complete the remaining!


cc: @illume , @joaquimrocha

**Comment by Pratik-050** [comment]
Hello @ChayanDass My PR is already up for it and waiting for review I think everything was fine with the PR just need some more people who can approve the changes as it is not so popular language so there is shortage of help around this. Feel free to give feedback though. Thanks!
